.github/workflows/push-tests.yml

```diff
@@ -39,6 +39,14 @@ jobs:
       - name: Lint frontend
         run: |
           docker-compose run --no-deps --rm frontend make lint
+      - uses: 8398a7/action-slack@v3
```

Comment:
Oooh this one is going to be super spammy no? It runs on every push to every non-merged PR in addition to main?